telegram/_files/sticker.py

```diff
@@ -52,7 +52,7 @@ class Sticker(_BaseThumbedMedium):
             position where the mask should be placed.
         file_size (:obj:`int`, optional): File size in bytes.
         bot (:class:`telegram.Bot`, optional): The Bot to use for instance methods.
-        **kwargs (obj:`dict`): Arbitrary keyword arguments.
+        _kwargs (:obj:`dict`): Arbitrary keyword arguments.
```

Comment:
ive seen this is many other places too, thought it's maybe ok to keep as is

Reply:
Yeah, I guess it doesn't matter too much since the parameter name is not actually used. This change was mainly due to me not realizing that the problem was the missing `:` in front of `obj` 😄